### DISCUSSION TITLE
Missing configuration parameter: setSameAppKey

### DISCUSSION THREAD

**Discussion by oren-oro** [admob-plus/admob-plus#428] [Ideas]
In the requestConfiguration there is a parameter which is called setSameAppKey:

https://developers.google.com/admob/ios/global-settings#same_app_key

This option is used to disable the request of the IDFA and prevent the need for the App Tracking Transparency consent message on iOS 14+.
This option is not available in the RequestConfig options of this plugin.

**Comment by ratson** [comment]
@oren-oro Added in latest release, you could set it via `admob.configure()`.
see https://admob-plus.github.io/docs/cordova/api#requestconfig

**Reply by oren-oro** [reply]
Thank you very much for this plugin and the quick response!
